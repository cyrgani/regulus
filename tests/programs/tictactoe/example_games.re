import(tictactoe),
import(random),

seed(1234),
play_game(),

print("first game over"),

seed(1234),
play_game(),
