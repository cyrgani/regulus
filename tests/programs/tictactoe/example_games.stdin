5
2
9
8
4
6
9
1
