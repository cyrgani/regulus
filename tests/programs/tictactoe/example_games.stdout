+-------+-------+-------+
|       |       |       |
|   1   |   2   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   4   |   5   |   6   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   7   |   8   |   9   |
|       |       |       |
+-------+-------+-------+
Please enter the number of the field you choose (1-9): 
+-------+-------+-------+
|       |       |       |
|   1   |   2   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   4   |   X   |   6   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   7   |   8   |   9   |
|       |       |       |
+-------+-------+-------+
+-------+-------+-------+
|       |       |       |
|   1   |   2   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   X   |   6   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   7   |   8   |   9   |
|       |       |       |
+-------+-------+-------+
Please enter the number of the field you choose (1-9): 
+-------+-------+-------+
|       |       |       |
|   1   |   X   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   X   |   6   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   7   |   8   |   9   |
|       |       |       |
+-------+-------+-------+
+-------+-------+-------+
|       |       |       |
|   1   |   X   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   X   |   6   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   8   |   9   |
|       |       |       |
+-------+-------+-------+
Please enter the number of the field you choose (1-9): 
+-------+-------+-------+
|       |       |       |
|   1   |   X   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   X   |   6   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   8   |   X   |
|       |       |       |
+-------+-------+-------+
+-------+-------+-------+
|       |       |       |
|   1   |   X   |   O   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   X   |   6   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   8   |   X   |
|       |       |       |
+-------+-------+-------+
Please enter the number of the field you choose (1-9): 
Game over, winner: X
+-------+-------+-------+
|       |       |       |
|   1   |   X   |   O   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   X   |   6   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   X   |   X   |
|       |       |       |
+-------+-------+-------+
first game over
+-------+-------+-------+
|       |       |       |
|   1   |   2   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   4   |   5   |   6   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   7   |   8   |   9   |
|       |       |       |
+-------+-------+-------+
Please enter the number of the field you choose (1-9): 
+-------+-------+-------+
|       |       |       |
|   1   |   2   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   X   |   5   |   6   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   7   |   8   |   9   |
|       |       |       |
+-------+-------+-------+
+-------+-------+-------+
|       |       |       |
|   1   |   2   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   X   |   O   |   6   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   7   |   8   |   9   |
|       |       |       |
+-------+-------+-------+
Please enter the number of the field you choose (1-9): 
+-------+-------+-------+
|       |       |       |
|   1   |   2   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   X   |   O   |   X   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   7   |   8   |   9   |
|       |       |       |
+-------+-------+-------+
+-------+-------+-------+
|       |       |       |
|   1   |   2   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   X   |   O   |   X   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   8   |   9   |
|       |       |       |
+-------+-------+-------+
Please enter the number of the field you choose (1-9): 
+-------+-------+-------+
|       |       |       |
|   1   |   2   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   X   |   O   |   X   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   8   |   X   |
|       |       |       |
+-------+-------+-------+
+-------+-------+-------+
|       |       |       |
|   1   |   O   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   X   |   O   |   X   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   8   |   X   |
|       |       |       |
+-------+-------+-------+
Please enter the number of the field you choose (1-9): 
+-------+-------+-------+
|       |       |       |
|   X   |   O   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   X   |   O   |   X   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   8   |   X   |
|       |       |       |
+-------+-------+-------+
Game over, winner: O
+-------+-------+-------+
|       |       |       |
|   X   |   O   |   3   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   X   |   O   |   X   |
|       |       |       |
+-------+-------+-------+
|       |       |       |
|   O   |   O   |   X   |
|       |       |       |
+-------+-------+-------+
